### PR TITLE
fix #75033 superhaber.tv

### LINKED ISSUE #75033: superhaber.tv
### Issue URL (Annoyance)
 [https://www.superhaber.tv/ekonomi](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.superhaber.tv%2Fekonomi)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#75033

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+superhaber.tv##.privacy-banner:not(body):not(html)
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: please add the domain to existing rule `##.privacy-banner`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:94]: Okay.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -4362,7 +4362,7 @@ eurosportplayer.com##.ply-notification
 shazam.com##.policy-label
 iddaa.com,ciceksepeti.com##.policy-popup
 decuro.de##.popupBase.notification
-ratemyprofessors.com##.preventAcceptance.balaMainContainer
+superhaber.tv,ratemyprofessors.com##.preventAcceptance.balaMainContainer
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/cookies_specific.txt:4365]: one line lower =)

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 2 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Ah yes, I overlooked.

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**